backend/satellite_tools/repo_plugins/apt_src.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright (c) 2021 SUSE LLC
+
+This software is licensed to you under the GNU General Public License,
+version 2 (GPLv2). There is NO WARRANTY for this software, express or
+implied, including the implied warranties of MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+along with this software; if not, see
+http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+SUSE trademarks are not licensed under GPLv2. No permission is
+granted to use or replicate SUSE trademarks that are incorporated
+in this software or its documentation.
+
+APT plugin for spacewalk-repo-sync.
+"""
+
+from spacewalk.satellite_tools.repo_plugins.deb_src import ContentSource as deb_ContentSource
+from spacewalk.satellite_tools.syncLib import log2
+from spacewalk.server import rhnSQL
+
+try:
+    #  python2
+    from urllib import unquote
+    import urlparse
+except ImportError:
+    #  python3
```

Comment:
Is Python2 compatibility needed?

Reply:
After discussion with @mcalmer decided to move the logic to `deb` plugin as additional plugin requre a lot of changes outside plugin itself.